latest/index.bs

```diff
@@ -175,7 +175,309 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
                 └── n
 </pre>
 
+Tables {#table-layout}
+----------------------
+The following describes the expected layout for tabular data.
+OME-NGFF tables are compatible with the [AnnData model](https://github.com/scverse/anndata).
 
+<pre>
+.                             # Root folder, potentially in S3,
+│                             # with a flat list of images.
+│
+└── 123.zarr
+    |
+    ├── .zgroup
+    |
+    ├── .zattrs
+    |
+    └── tables                # The tables group is a container which holds one or multiple tables that are compatible with AnnData.
+        |
+        │                     # The tables group MAY be in the root of the zarr file.
+        ├── .zgroup           # The tables group MAY be in root or in another group.
+        |
+        ├── .zattrs           # `.zattrs` MUST contain "tables", which lists the keys of the subgroups that are tables. In this case, the only table is "my_table".
+                              # hence `.zattrs` should be equal to `{ "tables": [ "my_table" ] }`.
+        |
+        └── my_table
+            │                     # The table group MAY be in the root of the zarr file.
+            ├── .zgroup           # The table group MAY be in root or in another group.
+            |
+            ├── .zattrs           # `.zattrs` MUST contain "type", which is set to `"ngff:region_table"`
```

Comment:
I just noticed that I have this as `ngff:regions_table` in [this example](https://deploy-preview-20--ome-ngff-validator.netlify.app/?source=https://minio-dev.openmicroscopy.org/idr/temp_table/test_segment.zarr/tables/regions_table)

Reply:
which are you suggesting needs fixing?

Reply:
I think it was due to a typo in our code, we should be careful to use `region` and `region_key` instead of `regions` and `regions_key`

Reply:
@kevinyamauchi I just noticed that I had forgotten to submit my review months ago (it was one of my first review with the GitHub interface and I have must forgotten to submit, sorry about that). I did it now and one of the comments was actually on `region` vs `regions`. I am now fine with both spellings.

Reply:
I'm trying to use https://github.com/kevinyamauchi/ome-ngff-tables-prototype/blob/0b7e59c58caf07e5f4e37756b396afe3e05e48e9/src/ngff_tables_prototype/reader.py#L307 to read a table, and this expects `"@type":  "ngff:points_table"`.
So there is a difference between `@type` and `type` but also the spec says it MUST be `ngff:region_table`. So is this simply too strict and we support various types here?

Reply:
Do we want to enumerate the "type" options that we support here?
E.g. `regions_table`, `points_table` or some other table that doesn't have any documented structure?